batch/data-plane/pom.xml

```diff
@@ -65,6 +65,19 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>com.microsoft.azure</groupId>
+      <artifactId>azure-mgmt-resources</artifactId>
+      <type>test-jar</type>
+      <scope>test</scope>
+    </dependency>
+
+    <dependency>
```

Comment:
is this redundant?

Reply:
No, 
Both are referencing to different versions of the azure-mgmt-resources in the parent pom.
Both the versions are needed.